planner/cascades/transformation_rules.go

```diff
@@ -591,3 +594,43 @@ func (r *PushSelDownJoin) OnTransform(old *memo.ExprIter) (newExprs []*memo.Grou
 	newJoinExpr.SetChildren(leftGroup, rightGroup)
 	return []*memo.GroupExpr{newJoinExpr}, true, false, nil
 }
+
+// SameProjectionElimination eliminate the projection.
+type SameProjectionElimination struct {
```

Comment:
Simplify the name to `ProjectionElimination`?

Reply:
Or `EliminateProjection`? We have been using names like `PushXXXDown`, `EnumeratePath`.

Reply:
OK, I will use `EliminateProjection`.